specification/synapse/resource-manager/Microsoft.Synapse/stable/2020-12-01/examples/Libraries_ListByWorkspace.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "subscriptionId": "12345678-1234-1234-1234-12345678abc",
+    "resourceGroupName": "exampleResourceGroup",
+    "workspaceName": "exampleWorkspace",
+    "ifNoneMatch": null,
+    "api-version": "2019-06-01-preview"
```

Comment:
We should also change the api-version in examples

Reply:
changed